timify-api/models/index.js

```diff
@@ -1,3 +1,8 @@
 import {Podcast} from './podcast.js'
+import { Episode } from './episode.js';
 
-export {Podcast};
\ No newline at end of file
+Podcast.hasMany(Episode, { foreignKey: 'podcastUuid' });
```

Comment:
should the foreignKey be podcastUuid and not something like episodeUuid? I don't know exactly what the API returns so just want to make sure.

Reply:
oh thanks for pointing that out. you're right.